esrally/client/asynchronous.py

```diff
@@ -202,22 +202,38 @@ def response(self, path):
 class RallyAiohttpHttpNode(elastic_transport.AiohttpHttpNode):
     def __init__(self, config):
         super().__init__(config)
-
-        self._loop = asyncio.get_running_loop()
-        self._limit = None
-
-        client_options = config._extras.get("_rally_client_options")
-        if client_options:
-            self._trace_configs = client_options.get("trace_config")
-            self._enable_cleanup_closed = client_options.get("enable_cleanup_closed")
-
-        static_responses = client_options.get("static_responses")
-        self.use_static_responses = static_responses is not None
-
-        if self.use_static_responses:
+        self._loop = None
+        self._trace_configs = None
+        self._enable_cleanup_closed = None
+        self._static_responses = None
+
+    @property
+    def trace_configs(self):
+        return self._trace_configs
+
+    @trace_configs.setter
+    def trace_configs(self, trace_configs):
+        self._trace_configs = [trace_configs]
+
+    @property
+    def enable_cleanup_closed(self):
+        return self._enable_cleanup_closed
+
+    @enable_cleanup_closed.setter
+    def enable_cleanup_closed(self, enable_cleanup_closed):
+        self._enable_cleanup_closed = enable_cleanup_closed
```

Comment:
Why don't we simply use an `enable_cleanup_closed` attribute? I don't understand how those getter/setter help.

Reply:
I agree, the getters/setters aren't required, I was just staying consistent with approach I first took with `static_responses`. 

Addressed in https://github.com/elastic/rally/pull/1669/commits/b99b77308894e0d28c9b824bc37ed94d62f041bd